core/extension_set.rs

```diff
@@ -32,19 +27,19 @@ pub fn setup_op_state(
 // but not added to "ext:core/ops" virtual module.
 /// Collects ops from extensions & applies middleware
 pub fn init_ops(
```

Comment:
I think I will factor out this function further - to have one for creating ops and another to compute an op macroware.